app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ApplicationRecord
   validates :email, uniqueness: true
   validates_format_of :email, :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i, :on => :create
 
+  devise :database_authenticatable, :registerable,
```

Comment:
Wtf cyrus, why wasnt that there in the first place? Besides that. Fire the guy that deleted this by copy pasting without checking.